Python.base

```diff
@@ -0,0 +1 @@
+FROM python:3.6
```

Comment:
might call this file something like `Dockerfile.py.base` / `Dockerfile.base.python` to make it look more like a dockerfile